### ISSUE TITLE
Bug with fetching real time updates from firestore

### ISSUE THREAD

**Issue by progand** [fiery-data/fiery-vue#2]
I've encountered a bug with realtime collection. Collection is called `applications`. 
The bug appears if there are at least 1 document in this collection and I add new document via firebase function or  firebase console.
So my scenario is:
1. open page
2. fiery-vue fetches existing records and everything looks fine so far
```applications: [{...}]```
3. I add new document to `applications` via firebase console
4. I see error in console and Vue inspector shows me my copputed property 
```applications: [{...}, undefined]```

And it should be ```applications: [{...}, {...}]```. But instead of 2 objects in array I have 1 object and `undefined`.

Here is my code:

```
<!-- Vue -->
    <script src="https://unpkg.com/vue/dist/vue.js"></script>
    <!-- Firebase -->
    <script src="https://www.gstatic.com/firebasejs/5.5.0/firebase-app.js"></script>
    <script src="https://www.gstatic.com/firebasejs/5.5.0/firebase-firestore.js"></script>
    <!-- vue-firestore -->
    <script src="https://unpkg.com/fiery-vue@0.7.0/dist/fiery-vue.js"></script>
    <script>
        // Initialize Firebase
        var config = {
            ...
        };
        var firebaseApp = firebase.initializeApp(config);
    </script>

<script>
        Vue.use(FieryVue)
        var firestore = firebaseApp.firestore(firebaseApp)
        // Disable deprecated features
        firestore.settings({
            timestampsInSnapshots: true
        })

        var vm = new Vue({
            el: '#app',
            fiery: true, // required to add this.$fiery to this component
            data() {
                const $fiery = this.$fiery;
                return {
                    limit: 50
                }
            },
            computed: {
                applicationsOptions() {
                    const { limit } = this // we have to reference these here for this to work
                    return {
                        query: applications => applications.orderBy('timestamp', 'desc').limit(limit),
                        key: 'id', 
                        exclude: ['id']                 
                    }
                },
                applications() {
                    return this.$fiery(firestore.collection('applications'), this.applicationsOptions, 'applications');                        
                }
            },
            methods: {
                
            }
        })
    </script>
```

If I remove `query` from options it still doesn't work properly.
Can anybody help? Thanks!

**Comment by ClickerMonkey** [comment]
I haven't experienced anything like this before, but ill look into it!

**Comment by ClickerMonkey** [comment]
I was able to reproduce the issue, thank you for logging this!

**Comment by ClickerMonkey** [comment]
Fixed in 0.7.2
